rebasehelper/cli.py

```diff
@@ -240,6 +248,17 @@ def __getattr__(self, name):
 class CliHelper(object):
 
     @staticmethod
+    def set_colors(app):
```

Comment:
I'd call this method differently, something like `colorize_output` or `should_use_colors`.